### PR TITLE
Support bulleted lists

### PR DESCRIPTION
**Pull request by bosschaert**
Fixes: #103

Test URLs:
- Before: https://main--zeiss--hlxsites.hlx.live/en/semiconductor-manufacturing-technology/news-and-events/smt-press-releases/2023/new-multifunctional-factory-in-wetzlar
- After: https://issue-103--zeiss--hlxsites.hlx.live/en/semiconductor-manufacturing-technology/news-and-events/smt-press-releases/2023/new-multifunctional-factory-in-wetzlar

### CODE REVIEW

**Comment by bosschaert** [comment]
Marking this as draft as the generated output is different in a seemingly unrelated area than the local rendering.

**Comment by bosschaert** [comment]
I force-pushed the original change as now it renders correctly. Not sure what caused the strange rendering initially.

**Comment by bosschaert** [comment]
If I click on the pagespeed link in the pagespeed report I get 100, so I assume the low LHS is something to so with the setup of obtaining it...

File: styles/simplified-styles.css [hunk 1]
```diff
@@ -46,6 +46,12 @@ a:hover {
     color: var(--button-primary-border-hover);
 }
 
+li {
+    list-style-type: disc;
+    margin: 0 2.5rem;
```
Comment (sdmcraft) [inline styles/simplified-styles.css:51]: I think the right side margin is not required. 

image

**Review by sdmcraft — APPROVED** [review]

File: styles/simplified-styles.css (hunk 1 above)
Comment (bosschaert) [reply styles/simplified-styles.css]: Thanks @sdmcraft I fixed that now

**Pull request merged**